src/app/index.spec.tsx

```diff
@@ -45,6 +45,24 @@ describe('create app', () => {
     expect(app.history.location.state).toEqual('asdf');
   });
 
+  describe('inside the browser', () => {
+    jest.unmock('../helpers/Sentry');
+    (global as any).window = window;
+
+    it('adds sentry middleware when enabled', () => {
+      jest.mock('../config', () => ({
```

Comment:
it'd be more explicit to use `doMock` here, `mock` has associated magic that you don't want in this case